src/sys/signal.rs

```diff
@@ -410,6 +410,50 @@ pub unsafe fn sigaction(signal: Signal, sigaction: &SigAction) -> Result<SigActi
     Errno::result(res).map(|_| SigAction { sigaction: oldact })
 }
 
+/// Signal management (see [signal(3p)](http://pubs.opengroup.org/onlinepubs/9699919799/functions/signal.html))
+///
+/// Installs `handler` for the given `signal`, returning the previous signal
+/// handler. `signal` should only be used following another call to `signal` or
+/// if the current handler is the default. The return value of `signal` is
+/// undefined after setting the handler with [`sigaction`][SigActionFn].
+/// 
+/// # Examples
+///
+/// ```no_run
+/// use nix::sys::signal::{Signal, SigHandler, signal};
+///
+/// assert_eq!(unsafe { signal(Signal::SIGINT, SigHandler::SigIgn) }.unwrap(), SigHandler::SigDfl);
+/// assert_eq!(unsafe { signal(Signal::SIGINT, SigHandler::SigDfl) }.unwrap(), SigHandler::SigIgn);
+/// ```
+///
+/// # Errors
+/// 
+/// Returns [`Error::UnsupportedOperation`] if `handler` is
+/// [`SigAction`][SigActionStruct]. Use [`sigaction`][SigActionFn] instead.
+/// `signal` also returns any error from `libc::signal`. such as when an attempt
```

Comment:
Can you put a blank line above this one? These should be separate paragraphs since they're distinct comments.